etc/p3-config/p3-config.yml

```diff
@@ -22,6 +23,12 @@ p3_project:
   keypairs: "{{ p3_keypairs }}"
   quotas: "{{ p3_quotas }}"
 
+octavia_project:
+  name: service
+  description: Project for Octavia service
+  project_domain: default
+  user_domain: default
```

Comment:
Kolla ansible should add this.

Reply:
Yes but I did not want p3-config requires this project to be present for adding security groups. There is no harm in having it here.